authn/service.go

```diff
@@ -10,9 +10,9 @@ import (
 )
 
 const (
-	loginDuration = 10 * time.Hour
-	resetDuration = 5 * time.Minute
-	issuerName    = "mainflux.authn"
+	loginDuration    = 10 * time.Hour
+	recoveryDuration = 5 * time.Minute
+	issuerName       = "mainflux.authn"
 )
```

Comment:
Can we use this as default values and provide env variables? IMHO 5min is to short for recovery token, I would set at least 15+ min